### ISSUE TITLE
A small mistake in tokenize_midi_dataset() I guess?

### ISSUE THREAD

**Issue by Fansesi** [Natooz/MidiTok#18]
Hi,
First of all I'd like to thank you @Natooz and others for your great work at this repository. It's such a life saver!

I'm playing around with the special tokens update and trying to convert a dataset into tokens using the `tokenize_midi_dataset()` method. In v1.3.1 it seems like you've changed `self.save_params(out_dir)` into `self.save_params(out_dir / 'config.txt')` under the  `tokenize_midi_dataset()`. But it throws the error below when I try to use it:

```
.../lib/python3.7/dist-packages/miditok/midi_tokenizer_base.py in tokenize_midi_dataset(self, midi_paths, out_dir, validation_fn, save_programs, logging)
    513         """
    514         Path(out_dir).mkdir(parents=True, exist_ok=True)
--> 515         self.save_params(out_dir / 'config.txt')  # Saves the parameters with which the MIDIs are converted
    516 
    517         for midi_path in tqdm(midi_paths, desc='Converting MIDIs to tokens') if logging else enumerate(midi_paths):

TypeError: unsupported operand type(s) for /: 'str' and 'str'
```

Am I wrong or should it be something like `self.save_params(out_dir + '/config.txt')`?

Thanks in advance!

**Comment by Natooz** [comment]
Hi, thank for your kind message, I'm always happy to help ! 😄

Thanks too for the bug report, indeed it would not work with an `out_dir` as `str`, that's fixed in f92f4aa98c4e407de9ca1d925e47b44833050aba !
It will be in the 1.3.2 update, that we release now (with this single fix) if it suits you better, otherwise just giving `out_dire` as a `Path` object (`from pathlib import Path`) would solve it: `Path(you_dir_as_str)`.
Sorry for this past negligence, have fun and reach out if you find any other bugs or improvements :)

**Comment by Fansesi** [comment]
Hi,
Thank you for your quick response and fixing the problem! I was giving my path as `str` just like you said, didn't think that giving it as `Path` object would solve the issue. I'm a quite new at using the `Path` library 😃.

Thanks again!
